src/Documentation/UserManual/DeviceInfraredSeekCam.dox

```diff
@@ -1,17 +1,15 @@
 /*!
 \page DeviceInfraredSeekCam Thermal Infrared Seek Pro camera
 
-\section DeviceInfraredSeekCamera 
+\section DeviceInfraredSeekCamera
 This device allow you to use a Infrared Seek Pro Camera to capture and send infrared thermal images.
 
 \image html DeviceInfraredSeekCam.png
 
-This device only works in Linux. We hope in the next months to integrate in Windows. 
-
 \section InfraredSeekCamConfigInstallation
 
-To use the camera you have to grant privileges to write to the camera. We have made a small script that you hace to execute with root privileges and you have to execute only just one time.
-You can found the instructions in:
+To use the camera you have to grant privileges to write to the camera. For Linux, we have made a small script that you hace to execute with root privileges and you have to execute only just one time. For Windows, the WinUSB driver is needed.
```

Comment:
Please provide a couple of details about what the script does. Root access scripts are a risk to users, so we might as well provide some info.